test/test_grids.jl

```diff
@@ -188,33 +188,35 @@ function test_regular_rectilinear_xnode_ynode_znode_and_spacings(arch, FT)
     for (grid_type, grid) in zip(grids_types, grids)
         @info "        Testing grid utils on $grid_type grid...."
 
-        @test xnode(2, grid, Center()) ≈ FT(π/2)
-        @test ynode(2, grid, Center()) ≈ FT(π/2)
-        @test znode(2, grid, Center()) ≈ FT(π/2)
+        CUDA.@allowscalar begin
+            @test xnode(2, grid, Center()) ≈ FT(π/2)
+            @test ynode(2, grid, Center()) ≈ FT(π/2)
+            @test znode(2, grid, Center()) ≈ FT(π/2)
```

Comment:
I don't think these use the GPU since the `grid` above is `CPU`?

Reply:
Maybe we don't need to test `xnode` separately from `xnodes` for GPU, what do you think?

Reply:
it's not a CPU grid -- I changed it